packages/server/src/arpa_reporter/lib/templateRules.json

```diff
@@ -7477,4 +7478,4 @@
       "ecCodes": false
     }
   }
-}
```

Comment:
I think this is a linux / vim thing. I didn't explicitly change this, and I cannot easily find a way to remove it.

Reply:
This is the correct ending – we want files to end in a blank line, so all good!